content/en/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

```diff
@@ -34,6 +34,36 @@ etcdctl del "" --prefix
 
 See the [etcd documentation](https://github.com/coreos/etcd/tree/master/etcdctl) for more information.
 
+### CNI configuration cleanup
+
+CNI plugins use the directory `/etc/cni/net.d` to store their configuration.
+The `kubeadm reset` command does not cleanup that directory. Backup the directory if needed.
+Then to remove it, execute the following command:
```

Comment:
> Then to remove it, execute the following command:

This statement leave me wondering whether remove it is a recommended operation or a required one. "to remove it" smells like I have an option, but is that true?

Reply:
the user can decide to clean the node completely. leaving the cni config is actually problematic if they decide to reuse the machine but switch to a different cni. i can add a note about that.